### PR TITLE
Resolves #1331

### LINKED ISSUE #1300: Cannot show result of phpbrew known
### Output

```bash
PHP Deprecated:  Creation of dynamic property PhpBrew\Console::$loader is deprecated in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/Application.php on line 117

Deprecated: Creation of dynamic property PhpBrew\Console::$loader is deprecated in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/Application.php on line 117
PHP Deprecated:  Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38

Deprecated: Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38
PHP Deprecated:  Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38

Deprecated: Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38
===> Fetching release list...
Downloading https://www.php.net/releases/index.php?json=1&version=8&max=100 via curl extension
Downloading https://www.php.net/releases/index.php?json=1&version=7&max=100 via curl extension
PHP Warning:  file_put_contents(/Users/kresnasatya/.phpbrew/php-releases.json): Failed to open stream: No such file or directory in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/src/PhpBrew/ReleaseList.php on line 129

Warning: file_put_contents(/Users/kresnasatya/.phpbrew/php-releases.json): Failed to open stream: No such file or directory in phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/src/PhpBrew/ReleaseList.php on line 129
Exception: Can't store release json file
Trace:

    0) PhpBrew\ReleaseList->save()
    1) PhpBrew\ReleaseList->fetchRemoteReleaseList(GetOptionKit\OptionResult)
    2) PhpBrew\Tasks\FetchReleaseListTask->fetch()
    3) PhpBrew\Command\KnownCommand->execute()
    4) call_user_func_array([PhpBrew\Command\KnownCommand, 'execute'], [])
    5) CLIFramework\CommandBase->executeWrapper([])
    6) CLIFramework\Application->run(['/opt/homebrew/bin/phpbrew', 'known'])
    7) PhpBrew\Console->runWithTry(['/opt/homebrew/bin/phpbrew', 'known'])
    8) require('phar:///opt/homebrew/Cellar/phpbrew/2.0.0/bin/phpbrew/bin/phpbrew')
```

### Expected Result

```
phpbrew known

7.0: 7.0.3, 7.0.2, 7.0.1, 7.0.0 ...
5.6: 5.6.18, 5.6.17, 5.6.16, 5.6.15, 5.6.14, 5.6.13, 5.6.12, 5.6.11 ...
5.5: 5.5.32, 5.5.31, 5.5.30, 5.5.29, 5.5.28, 5.5.27, 5.5.26, 5.5.25 ...
5.4: 5.4.45, 5.4.44, 5.4.43, 5.4.42, 5.4.41, 5.4.40, 5.4.39, 5.4.38 ...
5.3: 5.3.29, 5.3.28 ...
```

### Command

Here's the command I run: 

    phpbrew known

### Platform

OS:

    e.g. macOS 13.1 M1

phpbrew version: 2.0.0

I was set php version 8.2.0 as default version. I install that php with homebrew.

### LINKED ISSUE #1331: php deprecated creation of dynamic property warning
### Output

```
─➤  phpbrew list
PHP Deprecated:  Creation of dynamic property PhpBrew\Console::$loader is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/Application.php on line 117

Deprecated: Creation of dynamic property PhpBrew\Console::$loader is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/Application.php on line 117
PHP Deprecated:  Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38

Deprecated: Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38
PHP Deprecated:  Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38

Deprecated: Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38
* php-8.1.17
```

### Expected Result

```
actual list
```

### Command

Paste your phpbrew install command here (be sure to install with `--debug` flag, e.g. `phpbrew --debug install`)

    phpbrew list

### Build Log

    trying to run any phpbrew command i get the above code.  When doing any install there is also the same php deprecated messages.  I can't do a `brew install phpbrew@1.28` or anything other than the latest.

### Platform

OS:

    ─➤  sw_vers
    ProductName:            macOS
    ProductVersion:         13.2.1
    BuildVersion:           22D68

Running PHP: 
    ─➤  php -v
    PHP 8.2.4 (cli) (built: Mar 16 2023 16:46:52) (NTS)
    Copyright (c) The PHP Group
    Zend Engine v4.2.4, Copyright (c) Zend Technologies
        with Zend OPcache v8.2.4, Copyright (c), by Zend Technologies

Installing PHP:
    installing any version

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- [Resolves](https://github.com/phpbrew/phpbrew/commit/ff0735da06371f0cdf5414143ccf66b6ebb0f17c) https://github.com/phpbrew/phpbrew/issues/1331
- Resolves #1300.
- Upgrading the `corneltek/cliframework` to resolve dynamic properties issue. And the related commit is available [here](https://github.com/c9s/CLIFramework/commit/0e69e4d9d8ee6c8b5abb7ebf0cf846e47d93b656).
- Running the `composer update corneltek/cliframework` command and outputs are as follows:

```sh
$ composer update corneltek/cliframework
Loading composer repositories with package information
Updating dependencies
Lock file operations: 0 installs, 1 update, 0 removals
  - Upgrading corneltek/cliframework (3.0.x-dev 01cee2d => 3.0.x-dev 0e69e4d)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Downloading corneltek/cliframework (3.0.x-dev 0e69e4d)
  - Upgrading corneltek/cliframework (3.0.x-dev 01cee2d => 3.0.x-dev 0e69e4d): Extracting archive
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,12 +24,12 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.2.5 || <8.2.0",
+        "php": "^7.2.5 || <9.0",
```
Comment (theofidry) [inline composer.json:27]: Maybe:

```suggestion
        "php": "^7.2.5 || ^8.0",
```

to keep it with `^` and `~` annotations?

**Review by theofidry — APPROVED** [review]
May I suggest `Fix PHP 8.3 deprecations` for the title?

File: composer.json (hunk 1 above)
Comment (peter279k) [reply composer.json:27]: I think it should be `"php": "^7.2.5 || ^8.0"` .

**Review by c9s — APPROVED** [review]

**Comment by c9s** [comment]
go!

**Pull request merged**